src/main/java/org/stellar/sdk/requests/sorobanrpc/SorobanRpcErrorResponse.java

```diff
@@ -0,0 +1,19 @@
+package org.stellar.sdk.requests.sorobanrpc;
+
+import lombok.Getter;
+
+@Getter
+public class SorobanRpcErrorResponse extends RuntimeException {
```

Comment:
i expected this to be checked exception, forcing callers to handle it, what was the reason for coercing this to unchecked with `extends RuntimeException`?

Reply:
hmm, after more review, I don't think we want to model an rpc error response as an exception, rather it is conveyed as an attribute in the SorobanRpcResponse instead, thinking this SorobanRpcErrorResponse can be removed?

Reply:
After the discussion in https://github.com/stellar/java-stellar-sdk/pull/492#discussion_r1278707543, I think it would be better to directly return the result instead of SorobanRpcErrorResponse. What do you think?